docs/taquito_conversion_helper_functions.md

```diff
@@ -0,0 +1,26 @@
+---
```

Comment:
this should go into prepare provider's documentation as a follow up to what you can do with the PreparedOperation